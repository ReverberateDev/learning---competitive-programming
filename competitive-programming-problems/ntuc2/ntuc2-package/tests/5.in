500 500
145143282 527023655 934338551 773589024 980755659 405958256 528387536 859472097 860001687 546824945 34453943 503315909 566470112 71991646 904376879 74705162 662633667 180956444 949462671 621861343 231488369 997872071 348883332 361780969 774782385 870754220 344048012 485984448 883100253 1860175 731411835 271906419 566385120 91298535 67783930 987796480 843649727 752242397 351853827 999244509 606758925 159075409 834767975 887028850 625345012 171211413 745229384 858492572 118059571 711421103 12600479 738639916 395062939 313041371 253369015 987295078 773792657 140759559 700396464 723289219 903973520 496245846 865787133 691382319 127788196 366677918 147831526 720995851 430865135 298306794 816936199 42596652 727615783 246038520 160087759 612809882 559185430 738295227 627987970 410145234 465086489 263875802 713925944 791962437 776601506 932384816 242392667 647326423 356130231 526878538 451189037 518622816 40062576 612969196 392942528 333545518 144801148 973691276 352052988 461591937 848332957 780083157 645888796 228033294 226198567 697574182 631875633 201146749 26468741 683650577 315571869 944950251 226435508 322768604 532735898 740510313 348115398 106660355 467661745 835376251 581954913 572973236 793411 558802841 526080318 302942545 256570116 778314161 870022570 627983858 371296378 337033123 942935816 572934368 217665154 627860420 988056652 336373531 696529356 149688924 175144833 781668565 269950810 200933928 61039685 733886453 244694277 427329752 904510536 193476246 992602104 368059794 149684689 753696133 795359539 94382445 898384952 529002694 878565897 948985847 413543021 715006479 958477986 614177712 606279024 704619339 415786106 95245453 591998679 87788312 396905696 680446581 498751307 870269257 902613880 604753863 749910282 224952166 1059514 631721939 246101095 153996071 825367152 682732892 620275782 689642749 565413948 468569373 726825121 812391049 644966928 288808000 831105797 728518947 983842995 676003048 882023189 632036436 952550467 741578921 521569030 174648412 376433838 620163338 477127202 223671981 313431607 311948810 445956933 519787629 106300407 7541400 369541987 499760376 375469146 392428314 899699712 549636701 706146718 750079975 957607862 75707356 371687511 348232653 448489139 539768430 818951150 370722444 309438264 260213565 62021917 794662744 613633051 36425767 640094836 828447067 497918278 266560063 235613400 47118384 573375541 60487244 596260863 963635818 763934332 599865673 232491253 713643531 38094911 754894563 34625177 496154381 224534201 720578558 549716000 261592029 413959349 445509425 441645458 725159386 649290520 315011397 338859080 660868571 333118293 838601145 363602804 422817958 351447543 16760129 443451033 260777338 963876635 594911299 221871119 547917446 38936819 801630040 386497409 535896431 560994372 435537873 501199470 703591374 402481384 208783920 700890795 533804464 399579726 297983146 65297119 693385748 514832373 146092972 962491141 212086128 351760884 872689747 985502958 276748333 362834805 225422578 323361980 317842214 650631989 289342864 217725800 842422317 509813254 208708711 507807038 814249095 150366542 550848615 795466844 329516906 173822854 724912 173248822 827080326 186802363 883168796 753874709 454005369 796696847 474428338 325333651 589065561 683647801 135516980 570136096 149194543 12509441 573554322 561537153 716244086 529047493 506394485 869704523 807598752 225065208 650510261 481954156 853996689 794467485 796516835 333062563 976071458 960343504 852127168 11420931 424040863 938129725 759695556 961079156 600490058 467731367 175682200 335910962 602844886 103676003 493022620 179640908 488148204 127075802 562421090 960243494 483904169 241707019 351831345 84098807 593013534 592149766 586841731 709044044 384123087 624650098 283198343 312359811 250891317 862067463 861583264 22846801 282277948 313895189 287899619 90298685 987727557 829529340 798141341 928961314 393487141 157452466 81173707 542429873 950651557 736047424 130689697 449058797 133124352 457699653 359087108 594875309 149937895 830240625 971883535 315125277 221310080 84798219 18633874 840459581 312596963 97148524 662700686 411948896 273301916 542348162 423547066 933708719 646972204 306719822 50515969 546922148 881215880 850619750 53678082 405609615 900837272 406430324 419188111 163002901 957527737 169135285 988772654 68510379 693303100 990948218 887968155 189717791 712788197 52559018 518423108 89925648 930281836 66340158 866646988 677022321 213000447 482207684 676981896 491930122 57521342 747726046 355131141 706387447 915051838 238146496 168791680 928071463 389717255 435094708 633306988 742714590 155774230 754451353 459922416 125358899 789741916 845247072 5514854 463532435 705857340 825895310 711146974 324739388 210376209 653686974 449229977 485149123 500107686 933487900 41528974 964518799 367177404 40692202 684922344 863432283 904436616 863874815 201394565 467078072 112995889 569322818 707724665 462290824 540072790 940178228 584253321 250443908 793318023
1 631721939
1 612809882
1 518622816
1 413959349
1 887028850
1 858492572
2
2
2
1 631721939
2
1 18633874
2
1 887028850
1 546824945
2
1 496154381
2
2
1 573375541
2
2
2
1 860001687
2
1 898384952
2
2
1 201394565
1 467731367
1 859472097
1 932384816
2
2
1 201394565
2
2
2
2
2
2
2
2
2
1 860001687
2
2
1 266560063
1 572973236
2
1 949462671
2
1 282277948
2
1 424040863
1 949462671
2
2
1 368059794
2
1 297983146
1 496154381
1 742714590
1 226198567
2
1 18633874
1 10325202
2
1 749910282
2
2
1 801630040
1 208783920
2
1 859472097
2
1 348883332
1 333545518
2
1 208783920
1 348883332
2
1 263875802
1 226198567
2
1 828447067
2
1 348115398
1 423547066
1 829529340
1 795466844
1 565413948
2
2
1 870269257
2
1 736047424
1 833541342
2
1 833541342
1 500107686
2
2
1 514832373
2
1 546824945
2
1 217725800
2
1 399579726
1 266560063
1 899699712
2
2
2
2
1 796696847
1 491930122
2
1 870269257
2
1 693303100
1 573375541
1 171211413
1 348115398
1 683647801
1 898384952
2
1 333545518
1 363602804
1 934338551
2
1 413543021
2
2
2
2
2
1 389717255
2
1 87788312
1 801630040
2
2
1 616513530
2
2
2
2
1 140759559
1 720578558
1 683647801
1 125358899
2
1 773792657
2
1 987727557
2
1 174648412
2
2
2
2
2
1 713643531
1 221871119
2
2
1 532735898
2
2
2
2
1 865787133
2
1 12600479
2
1 496032389
2
2
1 697574182
2
2
2
1 561537153
1 828447067
2
1 677022321
2
2
2
1 997872071
2
1 958477986
2
2
1 514832373
2
1 171211413
1 423547066
1 1860175
2
1 12509441
1 676003048
2
2
2
1 12509441
2
2
2
2
1 987727557
2
1 155774230
2
2
2
1 781668565
1 518622816
2
2
2
1 336373531
2
2
1 424040863
1 853996689
1 546922148
1 297983146
2
2
1 317842214
1 781668565
1 612809882
1 532735898
2
1 363602804
1 546922148
2
1 887968155
2
2
2
1 850619750
1 561537153
2
1 405609615
2
2
1 644966928
2
1 125358899
2
2
1 467731367
1 306933024
2
1 899699712
1 882023189
2
1 261592029
2
1 727615783
2
1 1860175
2
1 773792657
2
1 263875802
1 175144833
1 628240535
1 38094911
1 317842214
2
2
2
2
1 706146718
2
2
1 61039685
1 521569030
2
1 712788197
1 562421090
2
1 12600479
1 351760884
2
1 210376209
2
1 526080318
2
1 101958368
2
2
2
1 915051838
2
1 392428314
2
1 413959349
2
1 528387536
2
1 750079975
1 942935816
2
1 90298685
2
1 325333651
2
1 837910977
2
2
1 332526718
2
2
1 298306794
1 904510536
1 175682200
1 840459581
1 676003048
2
1 794467485
2
2
1 988056652
1 713643531
1 858492572
2
2
2
1 932384816
2
1 746709709
2
2
1 368059794
2
1 890728135
2
1 700396464
1 581954913
1 298306794
1 86994813
2
2
1 549716000
2
2
2
2
1 351447543
1 942935816
2
2
2
1 506138141
1 825895310
1 845367800
2
1 405609615
1 52559018
2
1 526080318
2
1 503315909
1 753874709
1 501199470
1 193476246
1 235613400
1 890728135
1 546670581
1 749910282
2
1 174648412
2
2
1 559185430
1 746709709
2
2
1 501199470
1 485984448
2
1 693303100
1 224534201
2
2
1 677022321
2
1 948985847
2
1 549716000
2
2
1 87788312
1 565413948
2
1 903973520
2
2
1 796696847
1 306933024
1 962491141
1 242392667
1 887968155
1 413543021
1 38094911
2
1 348232653
1 573554322
1 804310844
2
2
1 798141341
2
1 90298685
1 140759559
1 175144833
1 884785046
1 1059514
2
1 881215880
1 253369015
1 948985847
2
2
2
2
2
2
1 700396464
2
2
2
2
1 640274632
1 535896431
1 84098807
1 351447543
2
2
2
1 980755659
1 427329752
1 59697512
1 688581809
1 485984448
2
1 534919004
2
1 118059571
1 406430324
2
1 302942545
2
2
2
2
2
2
1 782099190
2
2
1 149684689
2
2
2
1 344048012
1 612969196
2
1 991660434
1 179640908
1 833541342
1 338859080
1 649290520
2
2
1 51881528
1 612969196
2
2
1 928961314
2
2
1 97148524
1 831105797
2
2
2
1 789741916
2
2
1 753874709
2
1 963887921
1 193476246
