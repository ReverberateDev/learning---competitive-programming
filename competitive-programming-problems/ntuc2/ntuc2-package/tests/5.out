6
6
6
5
6
6
7
7
8
8
8
9
10
10
14
14
13
13
13
13
13
13
13
13
13
12
12
14
15
16
16
16
17
19
18
19
19
21
20
22
20
20
21
26
26
27
28
30
30
31
30
31
32
32
32
32
34
33
33
34
35
35
35
35
35
36
36
36
36
36
36
36
38
39
40
41
41
41
41
41
43
43
44
44
44
44
45
46
46
46
47
47
47
47
48
48
48
49
50
50
49
48
50
50
50
49
49
49
49
48
49
49
49
49
49
49
50
50
50
50
48
46
47
47
47
47
48
48
49
48
48
47
47
48
49
48
47
47
47
47
47
48
48
50
52
52
53
54
54
54
54
55
56
55
56
58
59
60
60
60
60
60
63
64
64
63
63
63
62
62
62
61
61
62
62
63
63
63
63
63
63
63
64
64
63
68
67
67
69
69
69
69
69
68
69
68
68
66
67
67
66
68
68
69
67
68
68
68
68
68
68
67
67
67
67
68
68
68
69
69
71
72
72
72
72
72
72
72
72
73
73
73
75
77
77
76
76
77
77
79
79
79
80
80
79
